aie_runtime_lib/AIE/chess_intrinsic_wrapper.cpp

```diff
@@ -25,5 +25,3 @@ extern "C" void llvm___aie___lock___acquire___reg(unsigned id, unsigned val) {
 extern "C" void llvm___aie___lock___release___reg(unsigned id, unsigned val) {
   release(id, val);
 }
-extern "C" void llvm___aie___event0() { event0(); }
-extern "C" void llvm___aie___event1() { event1(); }
```

Comment:
Perhaps you can just comment them out with a comment and a `\todo` for clarity and so they can be re-instantiated even more easily?